.mypy.ini

```
@@ -0,0 +1,6 @@
+[mypy]
+python_version = 3.8
+
+[mypy-unified_planning.grpc.generated.*]
+ignore_errors = True
+exclude = generated
```

I don't think the python version is needed and the `exclude` option may only be set in the global section (`[mypy]`), but I think we can remove it and just keep the `ignore_errors`.

```suggestion
[mypy]

[mypy-unified_planning.grpc.generated.*]
ignore_errors = True
```